config/initializers/stripe.rb

```diff
@@ -1 +1,17 @@
 Stripe.api_key = ENV["STRIPE_SECRET_KEY"]
+
+StripeEvent.configure do |events|
+  events.subscribe 'charge.succeeded' do |event|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.